examples/spark-kubernetes-operator/sparkapp.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: spark-app-v1
+  namespace: spark-cluster
+spec:
+  components:
+  - name: spark-workload-component
+    type: spark-workload
+    properties:
+      name: my-spark-app
+      namespace: spark-cluster
+      type: Scala
+      mode: cluster
+      image: "gcr.io/spark-operator/spark:v3.1.1"
+      imagePullPolicy: Always
+      mainClass: org.apache.spark.examples.streaming.JavaQueueStream
+      mainApplicationFile: "local:///opt/spark/examples/jars/spark-examples_2.12-3.1.1.jar"
+      sparkVersion: "3.1.1"
+      volumes:
+        - name: "test-volume"
+          hostPath:
+            path: "/tmp"
+            type: Directory
+      driverCores: 1
+      executorCores: 1
+      driverVolumeMounts:
+        - name: "test-volume"
```

Comment:
Dose this config mount a local path of node? Can we use a PVC?

Reply:
> Dose this config mount a local path of node? Can we use a PVC?

Yes, the demo currently supports the local fs of a node. Let me try PVC. Then give feedback.

Reply:
Hi, the [spec of the Spark application](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/api-docs.md) allows specifying any volume type supported by K8S. The value type of `spec.volumes` is `[]Kubernetes core/v1.Volume`. 

It means the spark application can mount a `PVC` typed volume, the example is [here](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/user-guide.md#mounting-volumes).

The addon is a spark (schedule) operator on Kubernetes. It scheduler spark applications, whether specifying volume config option or not, it depends on the business condition.